app/lib/shared/integrity.dart

```diff
@@ -35,8 +36,18 @@ class IntegrityChecker {
   IntegrityChecker(this._db, {int? concurrency})
       : _concurrency = concurrency ?? 1;
 
-  /// Runs integrity checks, and reports the list of problems.
-  Stream<String> check() async* {
+  /// Runs integrity checks, and reports the problems via a [Logger].
+  Future<void> verifyAndLogIssues() async {
+    var count = 0;
```

Comment:
Should we log that we initiate the integrity check?

Reply:
the neat_periodic_task wrapper does it, and also its status (successful or failing)